### PR TITLE
Added country name to searchable fields

### PR DESCRIPTION
**Pull request by benbacardi**

### CODE REVIEW

File: Elsewhen/Extensions/TimeZone.swift [hunk 1]
```diff
@@ -44,6 +48,12 @@ extension TimeZone {
                 return true
             }
         }
+        if let country = country {
+            let countryLower = country.lowercased()
+            if countryLower.contains(st) || countryLower.contains(st.replacingOccurrences(of: "_", with: " ")) {
```
Comment (dwrss) [inline Elsewhen/Extensions/TimeZone.swift:53]: We could check for both strings with a single iteration [contains(where:)](https://developer.apple.com/documentation/swift/array/2297359-contains)

File: Elsewhen/Helpers/TimeZoneFlags.swift [hunk 2]
```diff
@@ -486,3 +486,419 @@ let europeanUnionTimeZones: Set<String> = [
...
+    "Antarctica/Davis" : "Antarctica",
+    "Antarctica/DumontDUrville" : "Antarctica",
+    "Antarctica/Macquarie" : "Australia",
+    "Antarctica/Mawson" : "Antarctica",
+    "Antarctica/McMurdo" : "Antarctica",
+    "Antarctica/Palmer" : "Antarctica",
+    "Antarctica/Rothera" : "Antarctica",
+    "Antarctica/Syowa" : "Antarctica",
+    "Antarctica/Troll" : "Antarctica",
+    "Antarctica/Vostok" : "Antarctica",
```
Comment (dwrss) [inline Elsewhen/Helpers/TimeZoneFlags.swift:685]: Wouldn't these already be returned by the identifier search?

**Review by dwrss — CHANGES_REQUESTED** [review]

File: Elsewhen/Helpers/TimeZoneFlags.swift (hunk 2 above)
Comment (benbacardi) [reply Elsewhen/Helpers/TimeZoneFlags.swift:685]: Yes, you're right, and I've removed others (like Argentina has a load, etc) - I just missed these. I'll remove them!

File: Elsewhen/Extensions/TimeZone.swift (hunk 1 above)
Comment (dwrss) [reply Elsewhen/Extensions/TimeZone.swift:53]: My bad, I mis-parsed the types 🤦‍♂️

**Review by dwrss — APPROVED** [review]

**Pull request merged**